src/test/java/de/retest/ui/descriptors/DefaultRetestIdProviderTest.java

```diff
@@ -109,7 +109,6 @@ public void invalid_text_or_type_should_result_in_non_empty_id() throws Exceptio
 		when( identifyingAttributes.get( "type" ) ).thenReturn( "+" );
 		final String id = cut.getRetestId( identifyingAttributes );
 		assertThat( id ).isNotEmpty();
-		assertThat( id ).contains( "retestid" );
```

Comment:
What's wrong with this and the other asserts? For instance, in line 112, we want the ID to contain `"retestid"`, so why not assert the ID prefix?